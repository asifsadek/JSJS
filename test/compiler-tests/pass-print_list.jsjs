print_list([1,2,3,4]);

print_list([]);

print_list(["a", "b", "c"]);

print_list([true, false, false, false]);

print_list([[1,2], [2,3], [3,4]]);
