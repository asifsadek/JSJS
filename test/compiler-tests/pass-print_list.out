List [ 1, 2, 3, 4 ]
List []
List [ "a", "b", "c" ]
List [ true, false, false, false ]
List [ List [ 1, 2 ], List [ 2, 3 ], List [ 3, 4 ] ]
